.github/workflows/format-command.yml

```diff
@@ -0,0 +1,40 @@
+name: format-command
+on:
+  repository_dispatch:
+    types: [format-command]
+jobs:
+  format:
+    runs-on: ubuntu-latest
+    steps:
+      # Checkout the pull request branch
+      - uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.PAT }}
+          repository: ${{ github.event.client_payload.pull_request.head.repo.full_name }}
+          ref: ${{ github.event.client_payload.pull_request.head.ref }}
+
+      # Setup Python environment
+      - uses: actions/setup-python@v1
+
+      # Install formatting tools
+      - name: Install formatting tools
+        run: pip install black blackdoc flake8
+
+      # Run "make format" and commit the change to the PR branch
+      - name: Commit to the PR branch if any changes
+        run: |
+          make format
+          if [[ $(git ls-files -m) ]]; then
+            git config --global user.name 'actions-bot'
+            git config --global user.email '58130806+actions-bot@users.noreply.github.com'
+            git commit -am "[format-command] fixes"
+            git push
+          fi
+
+      - name: Add reaction
+        uses: peter-evans/create-or-update-comment@v1
+        with:
+          token: ${{ secrets.PAT }}
```

Comment:
Right, just continuing on the discussion from https://github.com/GenericMappingTools/pygmt/pull/645#discussion_r502401961.

> Ok, shall we try using tibdex/github-app-token then? The step by step instructions at https://github.com/peter-evans/create-pull-request/blob/master/docs/concepts-guidelines.md#authenticating-with-github-app-generated-tokens seem fairly straightforward. After we make the GMT-bot Github App, we can 'install' it here on GenericMappingTools/pygmt and at GenericMappingTools/gmt-feedstock to authenticate these bots.
> 
> _Originally posted by @weiji14 in https://github.com/GenericMappingTools/pygmt/pull/645#discussion_r502671461_

I'll see if I can set up a GMT-bot [Github App](https://docs.github.com/en/free-pro-team@latest/developers/apps/creating-a-github-app) without admin permission in the GMT organization. Might see a notification pop up later.

Reply:
Ok, I've made a [GenericMappingTools bot](), and installed it in the PyGMT repo:

![GMT bot in PyGMT repo](https://user-images.githubusercontent.com/23487320/97058972-ba2de580-15eb-11eb-9ecf-2fc98d347526.png)

Also trying to see if I can transfer the permissions of that bot to the GMT organization (might be a notification for that).

Reply:
```suggestion
          token: ${{ steps.generate-token.outputs.token }}
```

Reply:
Transfer is done.

Reply:
Cool, and now I can't see the APP setting anymore since I'm not a GMT admin :sweat_smile:. But that's ok, things should be fairly set up with https://github.com/GenericMappingTools/pygmt/pull/646#pullrequestreview-516046931 now. I have the secret key APP_PRIVATE_KEY locally, but that can be re-generated/updated on your side easily enough.